src/System.AppContext/src/System.AppContext.builds

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
+    <Project Include="System.AppContext.csproj">
```

Comment:
Looks like we haven't provided the facade for the default configuration and we should only need the netcore50 variant of this as netcore50 and netcore50aot are the same, so eliminate netcore50aot.